server/scripts/cleanup_accounts_with_no_details.py

```diff
@@ -0,0 +1,60 @@
+import warnings
+
+import structlog
+import typer
+from sqlalchemy import exc as sa_exc
+
+from server.db import db, init_database, transactional
+from server.db.models import Account
+from server.db.models import OrderTable as Order
+from server.settings import app_settings
+
+logger = structlog.get_logger(__name__)
+
+
+def get_accounts_with_missing_details():
+    return db.session.query(Account).filter(Account.details == None)
```

Comment:
db.session -> Account.query (is a shorthand)

Reply:
I couldn't get that to work initially, but maybe I automatically fixed that afterwards when I started calling init_db in the main.

Thanks for the tip!